forui/lib/src/widgets/accordion/accordion_controller.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/widgets.dart';
+
+/// A controller that controls which sections are shown and hidden.
+class FAccordionController extends ChangeNotifier {
+  /// The duration of the expansion and collapse animations.
+  final Duration animationDuration;
+
+  /// A list of controllers for each of the headers in the accordion.
```

Comment:
```suggestion
  /// The animation controllers for each of the sections in the accordion.
  @internal
```

Reply:
Apparently this is not allowed? `invalid_internal_annotation`

Only public elements in a package's private API can be annotated as being internal.